content/rs/databases/active-active/aa-planning.md

```diff
@@ -0,0 +1,82 @@
+
+Changes made from the admin console to an Active-Active database configuration only apply to the cluster you are editing. For global configuration changes across all clusters, use the `crdb-cli` command-line utility.
+
+## Memory limits
+
+Database memory limits define the maximum size your database can reach across all database replicas and [shards]({{<relref "rs/concepts/terminology#redis-instance-shard">}}) on the cluster. Your memory limit will also determine the number of shards you'll need.
+
+Besides your dataset, the memory limit must also account for replication, Active-Active overhead, and module overhead. These features can significantly increase your database size, sometimes increasing it by four times or more.
+
+Factors to consider when sizing your database:
+
+- **dataset size**: you want your limit to be above your dataset size to leave room for overhead.
+- **database throughput**: high throughput needs more shards, leading to a higher memory limit.
+- [**modules**]({{<relref "/modules/_index.md">}}): using modules with your database consumes more memory.
+- [**database clustering**]({{<relref "/rs/databases/configure/clustering.md">}}): spreading your data into shards across multiple nodes (scaling out) means you cannot disable clustering or reduce the number of shards later (scaling in).
+- [**database replication**]({{<relref "/rs/databases/configure/replication.md">}}): enabling replication doubles memory consumption
+- [**Active-Active replication**]({{<relref "/rs/databases/active-active/_index.md">}}): enabling Active-Active replication requires double the memory of regular replication, which can be up to four times (4x) the original data size.
+- [**database replication backlog**]({{<relref "/rs/databases/active-active/manage-aa#replication-backlog/">}}) for synchronization between shards. By default, this is set to 1% of the database size.
+- [**Active-Active replication backlog**]({{<relref "/rs/databases/active-active/manage-aa.md">}}) for synchronization between clusters. By default, this is set to 1% of the database size.
+
+It's also important to know Active-Active databases have a lower threshold for activating the eviction policy, because it requires propagation to all participating clusters. The eviction policy starts to evict keys when one of the Active-Active instances reaches 80% of its memory limit. 
+
+For more information on memory limits, see [Memory management with Redis Enterprise Software]({{<relref "/rs/clusters/optimize/node-memory.md">}}), [Memory and performance]({{< relref "/rs/concepts/memory-performance/">}}), or [Database memory limits]({{<relref "/rs/databases/configure/memory-limit.md">}}).
+
+## Networking
+
+Network requirements for Active-Active databases include:
+
+- A VPN between each network that hosts a cluster with an instance (if your database spans WAN).
+- At least two (but no more than five) participating clusters.
```

Comment:
maybe omit this :)
We already mentioned number if participating clusters above, so here it might be irrelevant